packages/eas-cli/src/commands/env/delete.ts

```diff
@@ -1,22 +1,23 @@
 import { Flags } from '@oclif/core';
+import assert from 'assert';
 import chalk from 'chalk';
 
 import EasCommand from '../../commandUtils/EasCommand';
 import {
-  EASEnvironmentFlag,
   EASNonInteractiveFlag,
   EASVariableScopeFlag,
+  EasEnvironmentFlagParameters,
 } from '../../commandUtils/flags';
 import { EnvironmentVariableEnvironment, EnvironmentVariableScope } from '../../graphql/generated';
 import { EnvironmentVariableMutation } from '../../graphql/mutations/EnvironmentVariableMutation';
 import { EnvironmentVariablesQuery } from '../../graphql/queries/EnvironmentVariablesQuery';
 import Log from '../../log';
 import { promptAsync, toggleConfirmAsync } from '../../prompts';
-import { promptVariableEnvironmentAsync } from '../../utils/prompts';
+import { formatVariableName } from '../../utils/variableUtils';
 
 type DeleteFlags = {
-  name?: string;
-  environment?: EnvironmentVariableEnvironment;
+  'variable-name'?: string;
+  'variable-environment'?: EnvironmentVariableEnvironment;
```

Comment:
I agree that using `variable-name` instead of `name` makes sense for `eas env:update`, but I'm wondering whether using just `name` for `env:delete` wouldn't be better 🤔 What do you think?

Reply:
I think it is preferable to have a consistent naming convention across all commands. `variable-name` and `variable-environment` are always used to find the variable, `name` and `environment` - to change it.